.changeset/khaki-glasses-raise.md

```diff
@@ -0,0 +1,48 @@
+---
+'astro': minor
+---
+
+## Integration Hooks to add Middleware
```

Comment:
A small note that this should ideally be `####` so the headings are properly aligned in the changeset.md, but it looks like we're already using `##` before, so not worth to do anything now I think 😄